aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -38,14 +38,20 @@ Set up a CI build for an ASP.NET Core app, then create a continuous deployment r
 [Azure Web App sandbox](https://github.com/projectkudu/kudu/wiki/Azure-Web-App-sandbox)  
 Discover Azure App Service runtime execution limitations enforced by the Azure Apps platform.
 
+::: moniker range=">= aspnetcore-2.0"
+
 ## Application configuration
 
-With ASP.NET Core 2.0 and later, three packages in the [Microsoft.AspNetCore.All metapackage](xref:fundamentals/metapackage) provide automatic logging features for apps deployed to Azure App Service:
+In ASP.NET Core 2.0 and later, the following NuGet packages provide automatic logging features for apps deployed to Azure App Service.
```

Comment:
ooops! I wrote "and" ... should be "or later."

Recommend a colon to introduce the list.